sympy/simplify/tests/test_sqrtdenest.py

```diff
@@ -183,11 +181,13 @@ def test_issue_5653():
         sqrt(2 + sqrt(2 + sqrt(2)))) == sqrt(2 + sqrt(2 + sqrt(2)))
 
 def test_issue_12420():
-    I = S.ImaginaryUnit
-    assert _sqrt_match(4 + I) == []
```

Comment:
This test fails now because it treats `I` as a square root, but I don't think we should be testing internal functions. The actual sqrtdenest still gives the correct results.